src/main/java/factionsplusplus/utils/StringUtils.java

```diff
@@ -38,6 +40,12 @@ public static OfflinePlayer parseAsPlayer(String playerName) {
         return null;
     }
 
+    public static OfflinePlayer parseAsPlayer(UUID playerUUID) {
```

Comment:
Just curious on your reason for putting it in `StringUtils`

Reply:
I think my original idea was it would be for parsing a string to a player but at some point I changed it to parsing a UUID to a player, it probably should go somewhere else. not really sure where though